.github/workflows/publish_charm.yaml

```diff
@@ -47,26 +121,42 @@ jobs:
           cp -rp $TEMP_DIR/. .
           rm -rf $TEMP_DIR
       - name: Install charmcraft
-        run: sudo snap install charmcraft --classic
-      - name: Log in to the Container registry
-        uses: docker/login-action@v2
-        with:
-          registry: ${{ env.REGISTRY }}
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          echo ${{ needs.get-issue-number.outputs.pr-number }}
+          sudo snap install charmcraft --classic
+      - name: Download image artifacts
+        run: |
+          echo $SHA
+          echo ${{ env.LAST_COMMIT_SHA }}
+          echo 
+          gh api \
+            --method GET \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /repos/canonical/${{ github.repository }}/actions/runs \
```

Comment:
Sorry, there's a typo in the code I created, the `canonical` in `/repos/canonical/${{ github.repository }}/actions/runs` is unnecessary, since the github.repository is owner + repo_name.